### ISSUE TITLE
Efidisk cannot parse disk size "528K"

### ISSUE THREAD

**Issue by RedPaes** [bpg/terraform-provider-proxmox#314]
**Describe the bug**
Clone a VM with Uefi & a efidisk fails. It canot parse the size string:

` Error: failed to decode HTTP GET response (path: nodes/pmx07/qemu/103/config) - Reason: cannot parse disk size "528K"
`
Looks like the parser for Disks is missing a "K" option: https://github.com/bpg/terraform-provider-proxmox/blob/b33a8ce6205a5d606a90b55e1b7aeb05e9fe4c50/proxmox/utils.go#L56

Didn't have a deep lool... But looks like it's related to: https://github.com/bpg/terraform-provider-proxmox/issues/86

**To Reproduce**

1. Create a Template VM with an EFI Disk as seen in the screenshot below. 
2. Copy the VM with Terraform provider 
3. Copy will work, but terraform throws an error: ` Error: failed to decode HTTP GET response (path: nodes/pmx07/qemu/103/config) - Reason: cannot parse disk size "528K"
`

```terraform
resource "proxmox_virtual_environment_vm" "windows_vm" {
  name        = var.vm_config.vm_name
  description = "Managed by Terraform"
  tags        = ["terraform", "win-10"]
  node_name = "${var.vm_config.pm_node}"
  clone {
    vm_id = "${var.vm_config.template_id}"
    retries = 3
  }
  agent {
    enabled = true
  }
  disk {
    datastore_id = var.vm_config.datastore_id
    file_format  = "raw" 
    interface    = "scsi0"
    size         = 80
  }
  disk {
    datastore_id = var.vm_config.datastore_id
    file_format  = "raw" 
    interface    = "efidisk0"
    size         = 1
  }
  cpu {
    cores = var.vm_config.cores
  }
  memory {
    dedicated = var.vm_config.ram_mb
  }


  network_device {
    bridge = var.vm_config.network_device
  }

  operating_system {
    type = "win10"
  }

 #}
```

**Expected behavior**
Should be able to parse efidisks.

**Screenshots**
See Efidisk: 512K
![image](https://user-images.githubusercontent.com/8821824/234103815-b8bf8db9-074f-4cf9-9000-d514483b2ef5.png)

**Comment by bpg** [comment]
I think this will be fixed in #301

**Comment by RedPaes** [comment]
Thank you for your quick response. 

I just had a look at the PR. 
As i understand there is no fix for that specific problem currently: https://github.com/bpg/terraform-provider-proxmox/blob/1ec47dc27411b71511621f9e3765b21aa1faf036/proxmox/utils_test.go#L31 

I'm not a go developer so maybe i miss something ;)

**Comment by bpg** [comment]
Ah, you're right. There was a discussion in https://github.com/bpg/terraform-provider-proxmox/issues/86#issuecomment-1502564708 where we decided to not support size < 1Gb, missing this use case.

**Comment by bpg** [comment]
Well, this actually should be fixed in the new #326 😅
